docker/dev/ubuntu/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Develop Orion Context Broker Ubuntu-Latest
```

Comment:
As general comment, I think that some piece of documentation (i.e. an .md file) explaining how to use the dokcer stuff would be needed, in a similar way a .md file explaining the valgrand stuff is already in the repository  (see https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/vagrant.md)